### PR TITLE
Adding translation for key population

### PR DESCRIPTION
**Pull request by jmbabazi**

### CODE REVIEW

File: configuration/messageproperties/messages_en.properties [hunk 1]
```diff
@@ -1048,6 +1052,7 @@ pihcore.genitalUlcer=Genital ulcer
 pihcore.withoutCondom=Without condom
 pihcore.analSex=Anal sex
 pihcore.homoSexual=Male homosexuality activity
+pihcore.homosexual=Men who have sex with other men
```
Comment (lnball) [inline configuration/messageproperties/messages_en.properties:1055]: @jmbabazi I would recommend that you change the code for pihcore.homosexual.  It's confusing to have another one with different text but uppercase S "homoSexual".  Maybe change to menSexWithMen.  Thanks.

Comment (mogoodrich) [reply configuration/messageproperties/messages_en.properties:1055]: Agree with @lnball

**Review by mogoodrich — CHANGES_REQUESTED** [review]
Agree with @lnball 's changes...

**Comment by lnball** [comment]
Sorry to be so picky, James.  I'll approve with the small change.  Then you'll also need to update the htmlform.

**Review by lnball — APPROVED** [review]

**Pull request merged**